src/rollup/rollup.ts

```diff
@@ -95,6 +95,9 @@ export async function rollupInternal(
 			await graph.pluginDriver.hookParallel('closeBundle', []);
 		},
 		closed: false,
+		async [Symbol.asyncDispose]() {
+			await this.close();
+		},
```

Comment:
As this feature at the moment is not widely supported, we should probably do what the [TypeScript docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-2.html#using-declarations-and-explicit-resource-management) state and polyfill the Symbol before using it, otherwise this will be a nice "undefined" property:

```js
Symbol.asyncDispose ??= Symbol("Symbol.asyncDispose");
```

I think this should be safe to do. Apparently, Node [supports at least the Symbols](https://github.com/nodejs/node/pull/48518) since [v20.4.0](https://nodejs.org/en/blog/release/v20.4.0#support-to-the-explicit-resource-management-proposal), but browsers do not and the polyfill is cheap.

Reply:
See also the tests